innopoints/models/account.py

```diff
@@ -3,10 +3,19 @@
 Also contains the function to load the user for the login manager."""
 
 from flask_login.mixins import UserMixin
+from sqlalchemy.dialects.postgresql import JSONB
 
 from innopoints.extensions import db, login_manager
 
 
+DEFAULT_NOTIFICATIONS = {
+    'innostore': 'off',
+    'volunteering': 'off',
+    'project_creation': 'off',
+    'administration': 'off',
+    'service': 'email',
```

Comment:
an enum would be great for both, the group and the mode (or whatever `['off', 'email', 'push']` is called)